workshopper.js

```diff
@@ -391,6 +391,8 @@ Workshopper.prototype._printUsage = function () {
 }
 
 Workshopper.prototype.getExerciseMeta = function (name) {
+  if (!name) return false
```

Comment:
very minor style nit: would you mind putting a newline before the `return` please?